src/ontodia/data/sparql/responseHandler.ts

```diff
@@ -336,27 +328,12 @@ function isLocalizedEqual(left: LocalizedString, right: LocalizedString) {
     return left.lang === right.lang && left.text === right.text;
 }
 
-export function getNameFromId(id: string): string {
-    const sharpIndex = id.indexOf('#');
-    if (sharpIndex !== -1) {
-        return id.substring(sharpIndex + 1, id.length);
-    } else {
-        const tokens = id.split('/');
-        return tokens[tokens.length - 1];
-    }
-}
-
-export function getLocalizedString(label?: RdfLiteral, id?: string): LocalizedString {
+export function getLocalizedString(label?: RdfLiteral): LocalizedString {
```

Comment:
I think `label` parameter should be required. Also, please change the return type to `LocalizedString | undefined` (to make it easier to enable strict null checking mode in the future).